### PR TITLE
DR-1418 Flow to export snapshot to a workspace

### PR DESCRIPTION
**Pull request by myessail**
snapshotexport

### CODE REVIEW

**Comment by tlangs** [comment]
Could you post screenshots of the change?

File: src/modules/theme.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ export default createMuiTheme({
       selection: '#99CCFF',
     },
     primary: {
-      main: '#81AB52',
+      main: '#81ab52',
```
Comment (tlangs) [inline src/modules/theme.js]: Any reason to un-capitalize the hex color here?

File: src/components/DetailViewHeader.jsx [hunk 2]
```diff
@@ -95,6 +104,15 @@ export class DetailViewHeader extends React.PureComponent {
                 canManageUsers={true}
               />
             )}
+            <Button className={classes.exportButton} variant="contained" color="primary">
+              <a
+                target="_blank"
+                rel="noopener noreferrer"
+                href={`${terraUrl}/#import-data?url=${URL_ENCODED_TDR_ROOT_URL}&snapshotId=${of.id}&snapshotName=${of.name}&format=snapshot`}
```
Comment (tlangs) [inline src/components/DetailViewHeader.jsx]: is there a reason we're breaking out the TDR root url and not just using `window.location.origin` here?

**Review by snf2ye — APPROVED** [review]
LGTM!

File: src/components/DetailViewHeader.jsx [hunk 3]
```diff
@@ -95,6 +104,15 @@ export class DetailViewHeader extends React.PureComponent {
                 canManageUsers={true}
               />
             )}
+            <Button className={classes.exportButton} variant="contained" color="primary">
```
Comment (samanehsan) [inline src/components/DetailViewHeader.jsx:105]: Could `variant` and `color` go into the `exportButton` class?

**Review by samanehsan — APPROVED** [review]
I just had a small style question, but otherwise lgtm! 🌤️

**Pull request merged**